wordpress-mobile/WordPress-iOS
# ULS: update Auth to show unified 2FA view in Site Address flow.

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/336
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/385

This updates Auth to show the unified 2FA view in the Site Address flow, and enables `unifiedSiteAddress` for development.

Bonus: since all the views SIWA uses have been updated, `unifiedApple` is also enabled for development.

(Unified Google is enabled on https://github.com/wordpress-mobile/WordPress-iOS/pull/14663.)

To test:

Site Address:
- Log in via Site Address, with a WP account that has 2FA enabled.
- Verify all the views in the flow are the unified ones.

SIWA:
- Log in/sign up with SIWA. Verify the unified views are displayed.

PR submission checklist:
